sympy/core/assumptions.py

```diff
@@ -573,3 +573,93 @@ def __init__(cls, *args, **kws):
             pname = as_property(fact)
             if not hasattr(cls, pname):
                 setattr(cls, pname, make_property(fact))
+
+
+class AssumptionsWrapperMeta(type):
+    """
+    Metaclass to give is_[...] attributes to AssumptionsWrapper
+    """
+    def __init__(cls, *args, **kws):
+        for fact in _assume_defined:
+            pname = as_property(fact)
+            setattr(cls, pname, make_wrapped_property(fact))
+
+
+def make_wrapped_property(fact):
+    def getit(self):
+        from sympy.assumptions import ask, Q
+        pred = getattr(Q, fact)
+        return ask(pred(self.expr), self.assumptions)
```

Comment:
This `getit` function needs to be as fast as possible. I definitely wouldn't want the delayed import inside this function. It's possibly also faster to spell out the handlers explicitly like:
```
def is_positive(self):
    return ask(Q.positive(self), self.assumptions)
```

Reply:
Agreed. I moved `AssumptionsWrapper` from core to sympy.assumptions.ask module.